### ISSUE TITLE
__line fails, if line contains single/double quotes

### ISSUE THREAD

**Issue by telmich** [ungleich/cdist#154]
Single quotes could work, but double quotes cannot. need to find a way to output the line securely without getting it modified

**Comment by winduptoy** [comment]
@telmich I'm having trouble with this one.

On remote machine, `/etc/TEST.txt`
```
Hello
Hello
dc_eximconfig_configtype='local'
Hello
Hello
```


```
echo __line /etc/TEST.txt --line "dc_eximconfig_configtype='local'" --state absent | cdist config --initial-manifest - -v myhost
```
leaves the file unchanged. Is there something I'm missing necessary to escape those single quotes?

**Comment by darko-poljak** [comment]
@winduptoy Can you test a fix from PR: https://github.com/ungleich/cdist/pull/594

**Comment by winduptoy** [comment]
@darko-poljak @telmich #594 works for me.

**Comment by matthijskooijman** [comment]
Seems like this is fixed and can be closed?

**Comment by asteven** [comment]
Just for reference. Issues like this one would also be reliably solved by #663.
